### ISSUE TITLE
DllNotFoundException on 2021.3.3f1

### ISSUE THREAD

**Issue by TheLazyIndianTechie** [gree/unity-webview#759]
**Environment**

**System**: MacBookPro 13" M1, 2020
**OS**: MacOS Monterey Ver 12.1
**Unity Editor**: 2021.3.3f1 

I tested everything and it seemed to be working on 2020.3.22f1. However, when I start a project with editor version 2021.3.3f1, I get a DLL Import error. Below is a screen for the same. I can't figure out what the issue is. I can switch the same back to the previous editor and it works perfectly fine. I've created my entire project in 2021. Can you help me with how to fix this?

Console Output:
```
DllNotFoundException: WebView assembly:<unknown assembly> type:<unknown type> member:(null)`

WebViewObject.Init (System.Action`1[T] cb, System.Action`1[T] err, System.Action`1[T] httpErr, System.Action`1[T] ld, System.Action`1[T] started, System.Action`1[T] hooked, System.Boolean transparent, System.Boolean zoom, System.String ua, System.Int32 androidForceDarkMode, System.Boolean enableWKWebView, System.Int32 wkContentMode, System.Boolean wkAllowsLinkPreview, System.Boolean separated) (at Assets/Plugins/WebViewObject.cs:364)

SampleWebView+<Start>d__3.MoveNext () (at Assets/Scripts/SampleWebView.cs:37)
<img width="1143" alt="DLLImportError" src="https://user-images.githubusercontent.com/86476092/147191120-429a96f6-7f30-479d-929b-89d8dba5911e.png">

UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at /Users/bokken/buildslave/unity/build/Runtime/Export/Scripting/Coroutines.cs:17)

```

**Comment by KojiNakamaru** [comment]
I've fixed the mac build in #761 to include arm64 (m1 mac), and updated binaries. Could you please try the latest?

**Comment by TheLazyIndianTechie** [comment]
I' busy till the 29th but will try and quickly test it. Thank you so much!

**Comment by TheLazyIndianTechie** [comment]
> I've fixed the mac build in #761 to include arm64 (m1 mac), and updated binaries. Could you please try the latest?

I can confirm it is now working fine in 2021.2.3f1 on an M1 MBP. Thank you! :)
